example/lib/main.dart

```diff
@@ -42,23 +42,21 @@ class _HomeState extends State<Home> {
         NativeUpdater.displayUpdateAlert(
           context,
           forceUpdate: true,
-          appStoreUrl: '<Your App Store URL>',
-          playStoreUrl: '<Your Play Store URL>',
-          titlePrefix: 'Upgrade',
-          description: '<Your description>',
-          updateButtonLabel: 'Upgrade',
-          closeButtonLabel: 'Exit',
+          // appStoreUrl: '<Your App Store URL>',
```

Comment:
Please do not commit commented-out lines, if you want to do an example please create another example and explain why removing or adding attributes will affect the example. Don't assume the user will guess what you mean.

Reply:
Ok, I will revise. I forgot to uncomment it after testing.

Reply:
Resolved in #10